int32 robot_id
string client_id
string shelf
int32 items
